rasa/nlu/utils/hugging_face/hf_transformers.py

```diff
@@ -59,12 +60,35 @@ def __init__(
         self,
         component_config: Optional[Dict[Text, Any]] = None,
         skip_model_load: bool = False,
+        language: str = None,
     ) -> None:
         super(HFTransformersNLP, self).__init__(component_config)
 
         self._load_model_metadata()
         self._load_model_instance(skip_model_load)
+        self.language = language
         self.whitespace_tokenizer = WhitespaceTokenizer()
+        self.character_tokenizer = CharacterTokenizer()
+
+    @classmethod
+    def load(
+        cls,
+        meta: Dict[Text, Any],
+        model_dir: Optional[Text] = None,
+        model_metadata: Optional["Metadata"] = None,
+        cached_component: Optional["Component"] = None,
+        **kwargs: Any,
+    ) -> "Component":
+        if cached_component:
+            return cached_component
+
+        return cls(meta, language=model_metadata.language)
+
+    @classmethod
+    def create(
+        cls, component_config: Dict[Text, Any], config: RasaNLUModelConfig
+    ) -> "Component":
+        return cls(component_config, language=config.language)
```

Comment:
I guess we need to keep the part about checking the language support 

``` 
        # Check language supporting
        language = config.language
        if not cls.can_handle_language(language):
            # check failed
            raise UnsupportedLanguageError(cls.name, language)

        return cls(component_config, language=config.language)
```

Reply:
suggestion accepted